apps/frontend/components/auth/admin-or-global-employee.tsx

```diff
@@ -0,0 +1,31 @@
+'use client';
+
+import { useAuthStore } from '@/components/context/auth-store';
+import type { ReactNode } from 'react';
+
+interface AdminOrGlobalEmployeeProps {
```

Comment:
```suggestion
interface AdminOrManager {
```

I would *really* like to employ the termin Manager instead of GlobalEmployee